pkg/power/platform/source/redfish.go

```diff
@@ -204,6 +204,11 @@ func NewRedfishClient() *RedFishClient {
 }
 
 func (rf *RedFishClient) IsSystemCollectionSupported() bool {
+	// goroutine for collecting power info from Redfish already exists
+	if rf.ticker != nil {
+		return true
```

Comment:
> just curious if the check is skipped then  IsSystemCollectionSupported is true?

We can consider "system collection (redfish) is supported" ( = returns `true` ) if `rf.ticker` is not `nil` because this function is the only place to set `rf.ticker`.

> but seems we didn't finish the ticker , will that cause any issue ?

This goroutine should be alive while kepler-exporter is alive. So I think it s not a problem.